app/views/courses/StudentAssessmentsView.coffee

```diff
@@ -0,0 +1,92 @@
+require('app/styles/courses/student-assessments-view.sass')
+RootComponent = require 'views/core/RootComponent'
+FlatLayout = require 'core/components/FlatLayout'
+api = require 'core/api'
+User = require 'models/User'
+Level = require 'models/Level'
+utils = require 'core/utils'
+
+StudentAssessmentsComponent = Vue.extend
+  name: 'student-assessments-component'
+  template: require('templates/courses/student-assessments-view')()
+  components:
+    'flat-layout': FlatLayout
+  props:
+    classroomID:
+      type: String
+      default: -> null
+  data: ->
+    courseInstances: []
+    levelSessions: []
+    classroom: null
+    levels: null
+    sessionMap: {}
+    playLevelUrlMap: {}
+    levelUnlockedMap: {}
+  computed:
+    backToClassroomUrl: -> "/teachers/classes/#{@classroom?._id}"
+  created: ->
+    # TODO: Only fetch the ones for this classroom
+    Promise.all([
+      api.users.getCourseInstances({ userID: me.id }).then((@courseInstances) =>)
+      Promise.all([
```

Comment:
This second `Promise.all` doesn't seem necessary. Only one promise in here, right?

Reply:
The next one below also has just one promise

Reply:
Whoops, yup, you're correct.